shared_libraries/orca_shared/recovery/test/unit_tests/test_shared_recovery.py

```diff
@@ -94,6 +94,12 @@ def test_post_entry_to_fifo_queue_no_errors(self):
                         }
                     ]
                 )
+                # Verifies that request_method is set properly to new_job or update_file
+                self.assertTrue(
+                    request_method.value == "new_job"
```

Comment:
Looks like we already have test to check request_method 

```
                self.assertEqual(
                    request_method.value,
                    queue_contents[0].message_attributes["RequestMethod"][
                        "StringValue"
                    ],
                )

```


So you can keep this code and I think we are good.